src/fluree/db/cache.cljc

```diff
@@ -22,10 +22,15 @@
      (let [runtime      (Runtime/getRuntime)
            max-memory   (.maxMemory runtime)
            total-memory (.totalMemory runtime)
-           effective-mb (if (graalvm/runtime?)
+           is-graalvm?  (graalvm/runtime?)
+           effective-mb (if is-graalvm?
                           (/ total-memory 1024.0 1024.0)   ; Case 2: GraalVM
-                          (/ max-memory 1024.0 1024.0))]   ; Case 1: JVM
-       (int (/ effective-mb 2.0)))
+                          (/ max-memory 1024.0 1024.0))    ; Case 1: JVM
+           cache-mb     (int (/ effective-mb 2.0))]
+       (log/info (str "Detected " (int effective-mb) "MB available memory"
+                      (when is-graalvm? " (GraalVM native-image)")
```

Comment:
```suggestion
                      (when graalvm? " (GraalVM native-image)")
```

Reply:
Addressed in 1c5ce962ddba8a02d91d9eadbad54f6f2b715577